### PR TITLE
Add rz-silhouette to cutter builds

### PR DESCRIPTION
**Pull request by wargio**
**Your checklist for this pull request**
- [ ] I've read the [guidelines for contributing](https://cutter.re/docs/contributing/code/getting-started.html) to this repository
- [ ] I made sure to follow the project's [coding style](https://cutter.re/docs/contributing/code/development-guidelines.html)
- [ ] I've updated the [documentation](https://cutter.re/docs/user-docs.html) with the relevant information (if needed)

**Detailed description**

Adds https://github.com/rizinorg/rz-silhouette to the cutter build

**Test plan (required)**

To test it, we need to use a server to see if it works.

### CODE REVIEW

**Comment by wargio** [comment]
The linux builds were stuck because there were no runners for ubuntu 18

File: dist/bundle_rz_silhouette.ps1 [hunk 1]
```diff
@@ -0,0 +1,19 @@
+$dist = $args[0]
+$cmake_opts = $args[1]
+$python = Split-Path((Get-Command python.exe).Path)
+
+if (-not (Test-Path -Path 'rz-silhouette' -PathType Container)) {
+    git clone https://github.com/rizinorg/rz-silhouette.git --depth 1 rz-silhouette
```
Comment (XVilka) [inline dist/bundle_rz_silhouette.ps1:6]: Does it have submodules? I thought it's a monorepo.

**Review by XVilka — APPROVED** [review]
LGTM with one minor thing

File: dist/bundle_rz_silhouette.ps1 (hunk 1 above)
Comment (wargio) [reply dist/bundle_rz_silhouette.ps1:6]: no submodules. i added a subproject so we can build it also on windows.

Comment (XVilka) [reply dist/bundle_rz_silhouette.ps1:6]: But why then you need these?
```
    git -C rz-silhouette submodule init
    git -C rz-silhouette submodule update
```

Comment (wargio) [reply dist/bundle_rz_silhouette.ps1:6]: oh, i have reused the code from libyara. let me remove that

**Review by XVilka — APPROVED** [review]

**Pull request merged**